ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__ci-4.20.yaml

```
@@ -28,7 +28,7 @@ resources:
 tests:
 - as: telcov10n-metal-single-node-hub-ztp
   cluster: build05
-  cron: '* 1 * * *'
+  cron: 0 22 1-7 * 5
```

Should we do something like this if we don't want it to run?
```suggestion
  cron: 0 22 31 2 5
```